app/controllers/v2/ManageUserController.scala

```diff
@@ -56,4 +60,47 @@ class ManageUserController @Inject() (
         } yield Ok(viewUser(userTab, userCaseTabs))
     }
 
+  def deleteUser(pid: String, activeSubNav: SubNavigationTab = ManagerToolsUsersTab): Action[AnyContent] =
+    (verify.authenticated andThen verify.mustHave(Permission.MANAGE_USERS)).async {
+      implicit request: AuthenticatedRequest[AnyContent] =>
+        for {
+          userCases <- casesService.getCasesByAssignee(Operator(pid), NoPagination())
+          user      <- userService.getUser(pid)
+        } yield {
+          if (userCases.results.nonEmpty) {
+            Ok(cannotDeleteUser(user))
+          } else {
+            Ok(confirmDeleteUser(user, removeUserForm))
+          }
+        }
+    }
+
+  def confirmRemoveUser(pid: String): Action[AnyContent] =
+    (verify.authenticated andThen verify.mustHave(
+      Permission.MANAGE_USERS
+    )).async(implicit request =>
+      removeUserForm
+        .bindFromRequest()
+        .fold(
+          errors =>
+            for {
+              user <- userService.getUser(pid)
+            } yield Ok(confirmDeleteUser(user, errors)), {
+            case true =>
+              for {
+                user <- userService.getUser(pid)
+              } yield {
+                val updatedUser = user.copy(deleted = true)
+                userService
+                  .updateUser(updatedUser, request.operator)
```

Comment:
Did we add a markDeleted endpoint in the backend? Ignore me if not

Reply:
We did, my bad I didn't notice this, I added a service and connector method to use markDeleted and call that in the controller now